layouts/partials/sponsor-banner.html

```diff
@@ -7,7 +7,7 @@
             </div>
 
             <!-- object tag enables links inside the SVG image -->
-            <object class="sponsor-img" type="image/svg+xml" data="{{ "/img/sponsors/all_sponsors.svg" | relURL }}">
+            <object class="sponsor-img" type="image/png" data="{{ "/img/sponsors/all_sponsors_2022.png" | relURL }}">
```

Comment:
Also I just looked over this properly but is there any reason not to use an `img` tag

Reply:
currently no but based on that html comment it'll be needed once we get a nice, modular SVG version going

Reply:
Yeah it was just me being lazy and changing "xml+svg" to "png" or something.
Apologies I sorta half-arsed that after failing for a few hours